app/l10n/locales/az/messages.json

```diff
@@ -23,7 +23,7 @@
   "J6ypuD": "AXTAR",
   "AiAMXv": "Hesabınız üçün konfiqurasiya {0} mövcud ola bilər:\\n\\n{1}\\n\\n Oxunub:?",
   "uyJsf6": "Haqqında",
-  "uQiTch": "Qeydiyyat",
+  "uQiTch": "Accept call",
```

Comment:
Many strings here change from translated back to English. Why?

Reply:
Because the same translation was used for multiple strings, it was an issue that we didn't notice before.